plotly_resampler/figure_resampler/figure_resampler.py

```diff
@@ -160,6 +162,15 @@ def show_dash(
         self._host = kwargs.get("host", "127.0.0.1")
         self._port = kwargs.get("port", "8050")
 
+        # check if self._port is already in use
+        with socket.socket(socket.AF_INET, socket.SOCK_STREAM) as s:
+            if s.connect_ex(('localhost', int(self._port))) == 0:
```

Comment:
Must this be `"localhost"`? maybe it's a tad more generic if this uses `self._host`